packages/kmod-5.10-nvidia/nvidia-tesla-tmpfiles.conf.in

```diff
@@ -0,0 +1,3 @@
+C /etc/drivers/nvidia-tesla-NVIDIA_VERSION.toml
```

Comment:
nit: I'd use the underscores around the replacement variables like `__NVIDIA_X__` consistently, so that the usage in files like `nvidia-tesla-path-env.in` is not ambiguous:
```suggestion
C /etc/drivers/nvidia-tesla-__NVIDIA_VERSION__.toml
```